articles/verifying-website-domains/index.md

```diff
@@ -0,0 +1,390 @@
+
+```python
+myResolver = dns.resolver.Resolver()
+myAnswers = myResolver.resolve('massiveinsights.com', "TXT")
+ans = [item for item in myAnswers]
+ans
+```
+
+Output:
+
+```python
+[<DNS IN TXT rdata: "google-site-verification=34Jb2Tx28xLGRll4nwbUOq6dyu-P_VOD3OcO9qqDTKo">, <DNS IN TXT rdata: "mscid=w87OZ8XPKbefuE8BURYoqwPFO1CdSFjEo5RIqw7JWEL/fy7vVSsP/Q6Slgu5MYk2ihGpG6SHuF70r+a7X9J8cw==">, <DNS IN TXT rdata: "v=spf1 ip4:72.139.55.134 include:spf.protection.outlook.com -all">]
+```
+
+"MX" for digital mailing information Input:
+
+```python
+myResolver = dns.resolver.Resolver()
+myAnswers = myResolver.resolve('massiveinsights.com', "MX")
+ans = [item for item in myAnswers]
+ans
+```
+
+Output:
+
+```python
+[<DNS IN MX rdata: 0 massiveinsights-com.mail.protection.outlook.com.>]
+```
+
+_Screenshots:_
+
+![dns-txt-mx](/engineering-education/verifying-website-domains/dns-ptr.jpg)
+
+![dns-ns](/engineering-education/verifying-website-domains/dns-ptr.jpg)
+
+`Whois`, a continuous recording registry for websites, is one of several databases that compiles and lists website information.
+
+Input:
+
+```python
+import whois
+whois_info = whois.whois("interviews-massiveinsights.works")
+print("Domain registrar:", whois_info.registrar)
+print("WHOIS server:", whois_info.whois_server)
+print("Domain creation date:", whois_info.creation_date)
+print("Expiration date:", whois_info.expiration_date)
+```
+Output:
+
+```python
+Domain registrar: None
+WHOIS server: None
+Domain creation date: None
+Expiration date: None
+```
+
+_Screenshot_:
+
+![whois-invalid](/engineering-education/verifying-website-domains/whois-invalid.jpg)
+
+Null values would be an indication that modifications were made during the development of the website.
+
+Most original websites will openly disclose this information as exemplified below.
+
+Input:
+
+```python
+import whois
+whois_info = whois.whois("massiveinsights.com")
+print("Domain registrar:", whois_info.registrar)
+print("WHOIS server:", whois_info.whois_server)
+print("Domain creation date:", whois_info.creation_date)
+print("Expiration date:", whois_info.expiration_date)
+```
+Output:
+
+```python
+Domain registrar: GoDaddy.com, LLC
+WHOIS server: whois.godaddy.com
+Domain creation date: 2012-06-04 23:22:58
+Expiration date: 2021-06-04 23:22:58
+```
+
+_Screenshot_:
+
+
+![whois-valid](/engineering-education/verifying-website-domains/whois-valid.jpg)
+
+### Next up, Windows PowerShell 
+
+_Make sure to right-click and select the `Run as Administrator` option before running PowerShell._
+
+Windows PowerShell is structured differently. There are subcategorized features to select specific content in a website language. It is essential to select and include `.Content` at the end of the command.
+
+As shown below, brackets are used to call a website with an API to sift through permissible website layers and must begin with `curl`.
+
+Input:
+
+```PowerShell
+(curl host.io/careers.massiveinsights.works).Content
+```
```

Comment:
This career page does not exist https://host.io/massiveinsights.works

Reply:
Must have been a typo.
Corrected in the next commit.
It was properly typed in the screenshot and the output.

[http://host.io/careers-massiveinsights.works
](http://host.io/careers-massiveinsights.works
)

I'll be more careful about that in other articles.